handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -256,6 +256,13 @@ public static boolean isAlpnSupported() {
     }
 
     /**
+     * Returns {@code true} if the used version of OpenSSL supports OCSP stapling.
+     */
+    public static boolean isOcspSupported() {
+      return version() >= 0x10002000L;
```

Comment:
@Scottmitch is right. OpenSSL's OCSP APIs involve a lot of machinery which we don't support. (`d2i_OCSP_RESPONSE`, etc.) BoringSSL has a simpler API which just blindly passes byte strings along. Since it's BoringSSL-specific, consumers are going to need BoringSSL-specific code anyway.

Re the version number, yeah we try to set the version number to align with OpenSSL as best we can, so right now we pretend to be 1.0.2.